assigner/config/versions.py

```diff
@@ -0,0 +1,48 @@
+import jsonschema
+import logging
+
+from assigner.config.schemas import SCHEMAS
+from assigner.config.upgrades import UPGRADES
+
+logger = logging.getLogger(__name__)
+
+class ValidationError(jsonschema.ValidationError):
+    pass
+
+
+def validate(config, version=(len(SCHEMAS) - 1)):
```

Comment:
You can put expressions for parameter defaults?

Reply:
Apparently so!